### PR TITLE
Add ru.yandex.Browser

### PR DESCRIPTION
**Pull request by fat0troll**
Yandex Browser is Chromium-based browser, created by Russian search giant Yandex.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** I've contacted Yandex via browser ticket support system, and they responded with generic reply about considering flatpak distribution some time in future.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by JakobDev** [comment]
bot, build ru.yandex.Browser

**Comment by JakobDev** [comment]
As you use a existing binary, please add a file called `flathub.json` with this content:
```json
{
    "only-arches": ["x86_64"]
} 
```
So it will not published for ARM.

File: ru.yandex.Browser.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>ru.yandex.Browser</id>
+  <launchable type="desktop-id">ru.yandex.Browser.desktop</launchable>
+  <name>Yandex Browser</name>
+  <developer_name>Yandex</developer_name>
+  <summary>The web browser from Yandex</summary>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>LicenseRef-proprietary=https://yandex.ru/legal/browser_agreement</project_license>
+  <url type="homepage">https://browser.yandex.ru</url>
```
[the remark this answers is not in the document]
Comment (fat0troll) [reply ru.yandex.Browser.metainfo.xml]: done this, thanks

**Comment by fat0troll** [comment]
I've fixed codecs issues found by myself last week, this PR ready for review.

**Comment by fat0troll** [comment]
bot, build ru.yandex.Browser

**Comment by fat0troll** [comment]
bot, build ru.yandex.Browser

File: ru.yandex.Browser.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        dest-filename: chromium-codecs-ffmpeg-extra.deb
+        only-arches: [x86_64]
+      - type: file
+        path: cobalt.ini
+      - type: file
+        path: ru.yandex.Browser.metainfo.xml
+      - type: file
+        path: ru.yandex.Browser.svg
+      - type: file
+        path: ru.yandex.Browser.desktop
```
Comment (JakobDev) [inline ru.yandex.Browser.yaml:112]: It looks like the RPM already contains a.desktop and a .appdata file. Why did you not use those?

Comment (fat0troll) [reply ru.yandex.Browser.yaml:112]: RPM file contains different identifiers in appdata file, and different executable name in desktop file. Looks like it's less effort to just have ready to use files in repo than patch those coming from RPMs.

Comment (hfiguiere) [reply ru.yandex.Browser.yaml:112]: no it's better to use the provided one. You can use rename commands to rename them. And `desktop-file-edit` to modify the desktop file.

Comment (fat0troll) [reply ru.yandex.Browser.yaml:114]: I've moved to provided .desktop file, but in conrary have no reason to move to upstream .appdata, especially as Flatpak version of it should contain Flathub-specific information.

For example: https://github.com/flathub/com.brave.Browser/blob/master/com.brave.Browser.metainfo.xml. There is a releases list, which doesn't apply to upstream appdata file.

Comment (hfiguiere) [reply ru.yandex.Browser.yaml]: >  I've moved to provided .desktop file, but in conrary have no reason to move to upstream .appdata, especially as Flatpak version of it should contain Flathub-specific information.

This is untrue. There is nothing specific for flatpak, ie a flatpak compliant .desktop works everywhere. If need there is `desktop-file-edit` to tweak the upstream provided.

Comment (fat0troll) [reply ru.yandex.Browser.yaml]: I'm talking about appdata file, not desktop file. Desktop file is replaced with upstream one already.

**Comment by hfiguiere** [comment]
bot, build ru.yandex.Browser

**Pull request closed without merging**